src/gc/gcinterface.h

```diff
@@ -41,6 +41,51 @@ typedef enum
     walk_for_loh = 3
 } walk_surv_type;
 
+// Arguments to GCToEEInterface::StompWriteBarrierResize and
+// GCToEEInterface::StompWriteBarrierInitialize. 
+struct WriteBarrierResizeArgs 
```

Comment:
I am not sure what flags will work best. It can be both. Feel free to do whatever makes most sense to you.